src/frontend/web_application/src/components/DropdownMenu/index.jsx

```diff
@@ -6,15 +6,24 @@ import './style.scss';
 
 export { withDropdownControl };
 
-const DropdownMenu = ({ className, ...props }) => (
-  <Dropdown className={classnames('m-dropdown-menu', className)} {...props} />
+const DropdownMenu = ({ className, hasTriangle, ...props }) => (
+  <Dropdown
+    className={classnames(
+      'm-dropdown-menu',
+      { 'm-dropdown-menu--has-triangle': hasTriangle },
```

Comment:
don't you think when we don't need the triangle thing, we just have use the simple Dropdown component and add some custom styles in parent component ?